README.md

```diff
@@ -1,39 +1,58 @@
-# RADAR-Kubernetes [![Build Status](https://travis-ci.org/RADAR-base/RADAR-Kubernetes.svg?branch=master)](https://travis-ci.org/RADAR-base/RADAR-Kubernetes)
-Kubernetes deployment of RADAR-base.
+# RADAR-Kubernetes [![Build Status](https://travis-ci.org/RADAR-base/RADAR-Kubernetes.svg?branch=dev)](https://travis-ci.org/RADAR-base/RADAR-Kubernetes)
+The repository contains Helm charts and other configuration needed to install RADAR-base on a Kubernetes cluster.
 
+## Prerequisites
+#### Kubernetes cluster 
+You need to have a working Kubernetes installation and there are 3 ways to have that:
+* From cloud providers such as: [AWS](https://aws.amazon.com/eks/), [GCP](https://cloud.google.com/kubernetes-engine/), [Azure](https://docs.microsoft.com/en-us/azure/aks/), etc
+* Install it on your own servers with: [Rancher](https://rancher.com/), [Kubespray](https://github.com/kubernetes-sigs/kubespray), [Kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/), [OpenShift](https://www.okd.io/), etc
+* Install it on your local machine with: [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/), [K3S](https://k3s.io/), etc
+
+If you're not using a cloud provider you need to make sure that you can [load balance](https://kubernetes.github.io/ingress-nginx/deploy/baremetal/), [expose  applications](https://kubernetes.io/docs/concepts/services-networking/connect-applications-service/#exposing-the-service) and provide [persistent volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
+
+#### DNS server
+Some applications are only accessible via HTTPS and it's essential to have a DNS server via providers like GoDaddy, Route53, etc
+
+#### SMTP Server
+RADAR-Base needs an SMTP server to send registration email to researchers and participants.
+
+#### Dockerhub account
```

Comment:
Where do we use the Dockerhub account details in this set-up?

Reply:
We don't need Dockerhub account anymore.